docs/ilm-setup.asciidoc

```diff
@@ -0,0 +1,189 @@
+== Manual index lifecycle management
+
+APM Server allows users to manually set up an index lifecylce managment (ILM) policy.
+This setup is made possible because of Elasticsearch's
+{ref}/using-policies-rollover.html#using-policies-rollover[index rollover] action.
+
+Follow the guide below to set up a custom ILM policy for span indices.
+You can repeat the actions for any other indices.
+
+. Set up the default index template
++
+If you haven't already, you'll need to set up the default index template.
+This is accomplished by running the <<setup-command,`setup --template`>> command.
++
+--
+[source,js]
+-----------------------
+./apm-server setup --template
+-----------------------
+// CONSOLE
+--
+
+. Create a policy for spans.
++
+Rollover actions are triggered when one or more criteria are met.
+In this example, we're creating a policy named `apm_span_policy`.
++
+This policy rolls data over to a new index after _1 day_, or _50gb_, whichever comes first.
+It also tells the old indexes to delete after _10 days_.
+All of these values can be customized to your specific needs.
++
+--
+[source,js]
+-----------------------
+PUT _ilm/policy/apm_span_policy
+{
+  "policy": {
+    "phases": {
+      "hot": {
+        "actions": {
+          "rollover": {
+            "max_age": "1d", <1>
+            "max_size": "50gb" <2>
+          }
+        }
+      },
+      "delete": {
+        "min_age": "10d", <3>
+        "actions": {
+          "delete": {}
+        }
+      }
+    }
+  }
+}
+-----------------------
+// CONSOLE
+<1> Rollover after _1 day_
+<2> Rollover after _50gb_
+<3> Delete old indexes after _10 days_
+--
+
+. Set up the ILM index template.
++
+To use the index lifecycle management policy created in the previous step,
+you need to specify it in the index template used to create the indices.
+The following template associates `apm_span_policy` with indices created from the +apm-{stack-version}-span-ilm+ template.
++
+--
+["source","js",subs="attributes"]
+-----------------------
+PUT _template/apm-{stack-version}-span-ilm
+{
+  "order": 2,
+  "index_patterns": ["apm-{stack-version}-span-*"], <1>
+  "settings": {
+    "number_of_shards": 1,
```

Comment:
Let's remove `number_of_shards` and `number_of_replicas` and leave those to the index template loaded by apm-server